app/services/solana_rpc_client.rb

```diff
@@ -39,4 +39,14 @@ def testnet_client
       cluster: Rails.application.credentials.solana[:testnet_urls].first
     )
   end
+
+  def network_client(network)
```

Comment:
I would add a comment to two methods above that they're deprecated - unless You're certain that they're not used anywhere and it's safe to remove them